src/main.cpp

```diff
@@ -144,9 +144,9 @@ void loop()
 
 int mapToRange400(int value, int min, int max)
 {
-  float multiplier = (value - min) * 1.0 / (max - min);
-
-  return min(max(round(multiplier * 400), 0), 400);
+  uint16_t finalval = (value-min)*400/(max-min);
```

Comment:
Could you change the name of the variable to `new`?

Reply:
And also put spaces between the operators and operands

Reply:
sure